### PR TITLE
[SW-235] Added a copy_from method to copy inputs instead of just passing by re…

### PR DESCRIPTION
**Pull request by bhung-bdai**
…ference to SE3

### CODE REVIEW

**Review by myeatman-bdai — APPROVED** [review]
There are some unit tests for this class, could we add a unit test for this function to `test_pose3d.py`

**Comment by myeatman-bdai** [comment]
Looks good other wise.

File: spatialmath/pose3d.py [hunk 1]
```diff
@@ -1934,6 +1934,27 @@ def Rt(
             t = np.zeros((3,))
         return cls(smb.rt2tr(R, t, check=check), check=check)
 
+    @classmethod
+    def CopyFrom(
```
Comment (myeatman-bdai) [inline spatialmath/pose3d.py:1938]: Unit tests

**Review by myeatman-bdai — CHANGES_REQUESTED** [review]

File: spatialmath/pose3d.py (hunk 1 above)
Comment (bhung-bdai) [reply spatialmath/pose3d.py:1938]: Added! Thanks for the notice

**Comment by bhung-bdai** [comment]
Added unit testing. Should be good to go!

**Comment by myeatman-bdai** [comment]
Thanks!

**Review by myeatman-bdai — APPROVED** [review]

**Pull request merged**